app/src/main/java/org/mozilla/fenix/FenixApplication.kt

```diff
@@ -626,6 +628,14 @@ open class FenixApplication : LocaleAwareApplication(), Provider {
         }
     }
 
+    private fun setCustomizeHomeMetrics(settings: Settings) {
```

Comment:
I think this should be moved into `setPreferenceMetrics`.  Currently this is a hack to try to race Glean before the ping is cleared.  In the future we're hoping to move all these set shared preferences out of the main thread.  It will be better if they are all in the same function.

Reply:
Updated!